examples/with-flow/types/next.js.flow

```diff
@@ -0,0 +1,41 @@
+declare module "next" {
+  declare type NextApp = {
+    prepare(): Promise<void>;
```

Comment:
We have some more public methods on `app`.

- `app.render(req, res, pathname: string, query: any)`
- `app.renderToHTML(req, res, pathname, query)`
- `app.renderError(err, req, res, pathname, query)`
- `app.renderErrorToHTML(err, req, res, pathname, query)`

https://github.com/zeit/next.js/blob/master/server/index.js#L152